src/BaGet/appsettings.json

```diff
@@ -1,5 +1,6 @@
 {
   "ApiKey": "",
+  "Port": "http://*:5000",
```

Comment:
Let's stay consistent with ASP.NET Core's terminology. I think `Urls` is a better config name since it better captures the functionality exposed here.